libgit/repo.go

```diff
@@ -23,6 +23,7 @@ import (
 const (
 	kbfsRepoDir         = ".kbfs_git"
 	kbfsConfigName      = "kbfs_config"
+	kbfsConfigNameTemp  = "._kbfs_config"
```

Comment:
I'm not sure this naming is a good idea; that's the format for macOS xattr metadata files on unsupported filesystems.

Reply:
Good point, but I don't think it matters.  Even if macOS has already created it, it means the kbfs_config file already exists, and so the TLF creation would have already happened.

That said, it would probably be better to name it something else, but now that there are testers out there already using this fliename, I don't think changing it is a good idea unless absolutely necessary.